build.xml

```diff
@@ -47,13 +47,13 @@
         e.g -Dmetdataio.path=/d1/python/METdataio/
 
         use -Dpython.env.path to define a path to METplotpy directory.
-        e.g -Dpython.env.path=/d1/python/METviewer_py3.6.3/
+        e.g -Dpython.env.path=/d1/python/METviewer_py3.10.4/
 
     example build command...
     ant -Dbuild.properties.file=$HOME/METViewer/webapp/metviewer/WEB-INF/classes/build-model-vxtest.properties \
          -Dlog4j2.file=$HOME/METViewer/webapp/metviewer/WEB-INF/classes/build-model-vxtest.log4j2.xml \
          -Ddb.management.system=mariadb -Dmetcalcpy.path=/d1/python/METcalcpy/ -Dmetplotpy.path=/d1/python/METplotpy/\
-         -Dmetdataio.path=/d1/python/METdataio/ -Dpython.env.path=/d1/python/METviewer_py3.6.3/ -Dcontext.path.suffix="-mariadb" clean all
+         -Dmetdataio.path=/d1/python/METdataio/ -Dpython.env.path=/d1/python/METviewer_py3.10.4/ -Dcontext.path.suffix="-mariadb" clean all
```

Comment:
@bikegeek, Python 3.10.4 contains a Critical CVE:
```
python 3.10.4 3.7.16 3.8.16 3.9.16 3.10.9 binary CVE-2022-37454 Critical /met/Python-3.10.4/libpython3.10.so
```
That's why we updated it from 3.10.4 to 3.10.18 in [METbaseimage v3.4.1](https://github.com/dtcenter/METbaseimage/blob/aa96f25c3829087e5b4520e90bf0fa632ad4cd94/Dockerfile#L17C1-L17C23).

Can/should this (and all) references to 3.10.4 be changed to 3.10.18?

Reply:
[METbaseimage 3.3.1](https://github.com/dtcenter/METbaseimage/blob/aa96f25c3829087e5b4520e90bf0fa632ad4cd94/Dockerfile#L17) updates from 3.10.4 to 3.10.18 to resolve the Python CVE.
[METbaseimage 3.4.1](https://github.com/dtcenter/METbaseimage/blob/v3.4.1/Dockerfile#L12) updates from 3.12.0 to 3.12.11 to resolve another Python CVE.

So the Python bugfix number is increased for BOTH versions. If you don't update to 3.10.18, I think the USAF will report back to us that the critical Python CVE is still present.